### PR TITLE
[oneDNN] Fuse conv2d+biasAdd/FusedBatchNorm+sigmoid+mul into fusedConv2D

### PR DESCRIPTION
**Pull request by sachinmuradi**
This PR fuses following patterns into FusedConv2D for oneDNN backend :
a) Conv2d + BiasAdd + Sigmoid + Mul
b) Conv2d + FusedBatchNorm/V2/V3 + Sigmoid + Mul

This fusion shows up to 15% improvement for models with above patterns.

### CODE REVIEW

**Comment by gbaned** [comment]
@sachinmuradi Can you please review this PR ? Thanks!

**Comment by sachinmuradi** [comment]
@gbaned I think you added my name by mistake :)  , I am the author of this PR, would need @penpornk or other person to review it

**Comment by gbaned** [comment]
> 

@sachinmuradi My apologies. 
@penpornk  Can you please review this PR ? Thanks!

**Comment by gbaned** [comment]
@penpornk Can you please review this PR ? Thank you!

**Comment by ashiqimranintel** [comment]
Any update on this PR?

**Review by ezhulenev — APPROVED** [review]

File: tensorflow/core/grappler/optimizers/remapper.cc [hunk 1]
```diff
@@ -2690,6 +2871,59 @@ Status AddFusedContractionNode(
   return OkStatus();
 }
 
+Status FuseConv2DSwish(RemapperContext* ctx,
+                       const std::map<string, int>& matched_nodes_map,
+                       const std::set<int>& remove_node_indices,
+                       std::vector<bool>* invalidated_nodes,
+                       std::vector<bool>* nodes_to_delete) {
+  const GraphDef* graph = ctx->graph_view.graph();
```
Comment (penpornk) [inline tensorflow/core/grappler/optimizers/remapper.cc:2879]: Could you please help remove the unused variable? Thank you!

**Review by penpornk — CHANGES_REQUESTED** [review]

File: tensorflow/core/grappler/optimizers/remapper.cc (hunk 1 above)
Comment (sachinmuradi) [reply tensorflow/core/grappler/optimizers/remapper.cc]: Done.

**Review by penpornk — APPROVED** [review]
Thank you!

**Pull request merged**